src/amber/support/dsls.cr

```diff
@@ -23,4 +23,24 @@ module Amber::Support::DSL
 
     {% end %}
   end
+
+  module ControllerActions
+    macro render_both(filename, layout)
```

Comment:
Perhaps. Because of the need to pass in the current file on render it's a little bit hard to get the exact same functionality. I don't feel that render_both will be needed very much.